project/src/main/java/com/google/sps/data/Places.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * The package that currently holds all the java files of the food-mood project.
+ */
+package com.google.sps.data;
+
+import com.google.appengine.repackaged.com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * A utility class for Place objects.
+ */
+public final class Places {
+
+  /**
+   * Sorts the given list of Places randonly.
+   * @param places the list we want to sort.
+   * @return the randomly sorted list.
+   */
+  public static ImmutableList<Place> randomSort(ImmutableList<Place> places) {
+    List<Place> mutablePlaces = new ArrayList<>(places);
+    Collections.shuffle(mutablePlaces);
+    return ImmutableList.copyOf(mutablePlaces);
+  }
+
+  /**
```

Comment:
no need for javadoc here (it's clear why it has a private constructor & private methods don't really require javadocs as they're not part of the public API).

If the linter obligates you to do so: add a rule that avoids requiring javadocs for private methods (and better yet, constructors).

Reply:
done